docs/github-labels.md

```diff
@@ -0,0 +1,108 @@
+# GitHub Label List
+
+We use GitHub labels to perform issue triage, track and report on development
+progress, plan roadmaps, and automate issue grooming.
+
+To ensure that contributing new issues and PRs remains straight-forward, we would
+like to keep the labels required for submission to a minimum. The remaining
+labels will be added either by automation or manual grooming by other
+contributors and maintainers.
+
+This labels in this list originated within Kubernetes at
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md.
+
+## Labels that apply to issues or PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| api-review                         | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                           | Issues or PRs related to an API | Any |
+| area/blog                          | Issues or PRs related to blog entries. | Any |
+| area/dependency                    | Issues or PRs related to dependency changes | Any |
+| area/endpoint/identity             | Issues or PRs related to endpoint identity | Any |
+| area/endpoint/selection            | Issues or PRs related to endpoint selection | Any |
+| area/endpoint/type                 | Issues or PRs related to endpoint type | Any |
+| area/ipam                          | Issues or PRs related to IP address management (IPAM) | Any |
+| area/licensing                     | Issues or PRs related to Antrea licensing | Any |
+| area/monitoring/logging            | Issues or PRs related to logging | Any |
+| area/monitoring/auditing           | Issues or PRs related to auditing | Any |
+| area/monitoring/traffic-analysis   | Issues or PRs related to traffic analysis | Any |
+| area/monitoring/mirroring          | Issues or PRs related to mirroring | Any |
+| area/network-policy/action         | Issues or PRs related to network policy actions | Any |
```

Comment:
network-policy/api/lifecycle/match/scope are probably the only labels which would make sense as of now?

Reply:
We are planning work in the next two milestones that will involve monitoring, etc. I'd like to begin creating issues for planned work 6-12 months out. We will display these on a roadmap project board.  The planned work includes all the areas listed.

After initial construction of the issues and board, let's remove any labels not used. Also, @jianjuns raised some valid points about areas tracking components or features. I'm not opposed to using both because the sorting and extraction of issues by component or feature serve different purposes from a project management perspective.

Additionally, we are not expecting contributors to apply all these labels up front (although it would be helpful if they did). I regular triage process should allow for additional classification of issues.